swssconfig/sample/00-copp.config.json

```diff
@@ -55,5 +55,21 @@
             "red_action":"drop"
         },
         "OP": "SET"
+    },
+    {
+        "COPP_TABLE:trap.group.sflow": {
+            "trap_ids": "sample_packet",
+            "trap_action":"trap",
+            "trap_priority":"1",
+            "queue": "2",
+            "meter_type":"packets",
+            "mode":"sr_tcm",
+            "cir":"1000",
+            "cbs":"1000",
+            "red_action":"drop",
+            "genetlink_name":"psample",
+            "genetlink_mcgrp_name":"packets"
+        },
+        "OP": "SET"
```

Comment:
suggest to remove this configuration, so that we can merge other parts of the PR.

Reply:
Hi Guohan,
Now the changes are made in such a way that only when "config sflow enable" is issued in platform, these configurations would be played. Hence in platforms which doesn't support sflow, these configurations would not be played at all